src/javase02/task02/Chancellery.java

```diff
@@ -0,0 +1,44 @@
+package javase02.task02;
+
+public class Chancellery {
+    int price;
+    int number;
+}
+
+class Pen extends Chancellery{
```

Comment:
Empty class seems strange. Maybe better create class Pen and Enum with colors?

Reply:
or extend RedPen, GreenPen etc. directly from  Chancellery

Reply:
I made them abstract. Then i added isUsesInk and isUsesGraphite for both of them.